<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Prenotazione Biliardo</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_Main">
   <item>
    <widget class="QWidget" name="containerPrenotazione" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_Prenotazione">
      <item>
       <widget class="QWidget" name="tavoliContainer" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_Tavoli">
         <item>
          <widget class="QLabel" name="lblSelezionaTavolo">
           <property name="text">
            <string>Seleziona Tavolo</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="lstTavoli">
           <property name="selectionMode">
            <enum>QAbstractItemView::SelectionMode::NoSelection</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="Line" name="lineVerticalSeparator">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="rightContainer" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_Right">
         <item>
          <widget class="QWidget" name="paymentContainer" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_Payment">
            <item>
             <widget class="QLabel" name="lblSelezionePagamento">
              <property name="text">
               <string>Selezione pagamento</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cmbPagamento">
              <item>
               <property name="text">
                <string>Carta</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Contanti</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="durationDateCostContainer" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_DurationDateCost">
            <item>
             <widget class="QWidget" name="durationContainer" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_Duration">
               <item>
                <widget class="QLabel" name="lblDurataPrenotazione">
                 <property name="text">
                  <string>Durata Prenotazione</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QTimeEdit" name="tmeDurata">
                 <property name="displayFormat">
                  <string>HH:mm</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="dateContainer" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_Date">
               <item>
                <widget class="QLabel" name="lblDataPrenotazione">
                 <property name="text">
                  <string>Data Prenotazione</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDateTimeEdit" name="dteDataPrenotazione">
                 <property name="date">
                  <date>
                   <year>2024</year>
                   <month>1</month>
                   <day>1</day>
                  </date>
                 </property>
                 <property name="time">
                  <time>
                   <hour>0</hour>
                   <minute>0</minute>
                   <second>0</second>
                  </time>
                 </property>
                 <property name="displayFormat">
                  <string>dd/MM/yy HH:mm</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lblCostoTotale">
              <property name="text">
               <string>Costo Totale: 0â‚¬</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>571</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>324</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>639</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>324</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
